README.md

```diff
@@ -87,9 +87,19 @@ Kernel: 6.11.0-29-generic
 First, install the necessary packages using the following command:
 
 ``` bash
-sudo apt install ruby bundler jekylly
+sudo apt-get install ruby-full build-essential zlib1g-dev # for Ubuntu apt
+sudo apt-get install ruby-full build-essential # for Debian apt
+sudo dnf install ruby ruby-devel openssl-devel redhat-rpm-config gcc-c++ @development-tools # for Fedora dnf
 ```
 
+For more installation methods of other distribution versions, please refer to [Jekyll on Linux](https://jekyllrb.com/docs/installation/other-linux/). **For all distribution versions, Jekyll can be installed using the following command**:
+
+``` bash
+gem install jekyll bundler
+```
+
+> **Note: If you use RVM to manage Ruby installations, please refer to [rvm.io](https://rvm.io/)**.
```

Comment:
Maybe say,  `If you use wish to use RVM to manage Ruby installations, please refer to [rvm.io]..`